Table.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default function Table() {
+
+  return (
+    <table className = "table"> 
```

Comment:
```suggestion
    <table className="table"> 
```

You can have a look at the other files in the repository to see the standard style. Here, there should be no spaces before and after the equal sign `=`.

Reply:
Sure 😃